getchar.c

```diff
@@ -0,0 +1,10 @@
+#include <stdio.h>
+ /* verifying getchar output */
+int  main()
+{
+int c;
+if(c = getchar() != EOF){
+  printf("%d\n", c);
+  }
+return 0; 
```

Comment:
I find that the indentation is not consistent. In this gethcar program, you are using two spaces for indentation, while in the previous program, you are using 4 spaces for indentation. Make that spacing is consistent in the future. You can use `tab` for indentation and it should automatically be 2 spaces.